kolibri/plugins/facility/assets/src/views/FacilityConfigPage/ChangePinModal.vue

```diff
@@ -52,13 +54,14 @@
         } else {
           if (this.pinPattern.test(this.pin)) {
             this.pinError = '';
+            this.setPin({ pin_code: this.pin });
             this.$emit('submit');
+            this.showSnackbarNotification('pinUpadeted');
```

Comment:
small typo, this should be `pinUpdated`.

This is working correctly, because here and in the snackbar, it is spelled this way, so this should be changed in both places